themes/tiddlywiki/vanilla/base.tid

```diff
@@ -333,16 +333,17 @@ button svg, button img {
 }
 
 .tc-btn-boxed {
-	font-size: 0.6em;
```

Comment:
This css fixes only the new button styles as their styles were basically wrong, breaking the text size. Take the current TiddlyWiki, try to apply the new button styles while looking at the button text... and you'll see miniature text. That is, those buttons styles where designed without ever looking at a "button + text" combination. So, now both works.